Sources/TuistSupport/Utils/Environment.swift

```diff
@@ -77,6 +78,8 @@ public class Environment: Environmenting {
 
     /// Sets up the local environment.
     public func bootstrap() throws {
+        // load `.env` variables into the current process
+        try Dotenv.configure()
```

Comment:
Reading a .env file should be literally 10-20 lines of code, I wouldn't add an external dependency for that.

For example, looks like this configure would throw if the file is not found, which is not what we want

Reply:
I can change it to be `try? Dotenv.configure() in that case` but it did seem silly to add the code to `tuist` because I had already written a package that does exactly what I wanted prior to all of this. Glad to bring it in if others feel strongly about it.

Reply:
`try?` sounds good – but we probably do we want to log a warning in such a case.

Reply:
The fewer external dependencies we have the better, in particular if the dependency just saves a few line.

Apart from having a dependency, looking at this code I personally don't think it's clear what `Dotenv.configure()` would do (indeed you needed to add a comment to make it clear), and I would prefer a `self.loadEnvFile()` or something like that.

Anyway, I will leave the last word to @fortmarek.

Reply:
Sounds great, I'll make those changes as soon as I get time

Reply:
@kwridan, what's the recommended way to log an error for Tuist, is there a built in system?